docs/src/catalyst_for_new_julia_users/intro_to_catalyst.md

```diff
@@ -0,0 +1,182 @@
+# Introduction to Catalyst and Julia for new Julia users
+The Catalyst tool for the modelling of chemical reaction networks is based in the Julia programming language. While experience in Julia programming is advantageous for using Catalyst, it is not necessary for accessing some of its basic features. This tutorial serves as an introduction to Catalyst for those unfamiliar with Julia, also introducing some basic Julia concepts. Anyone who plans on using Catalyst extensively is recommended to familiarise oneself more thoroughly with the Julia programming language, e.g. by reading [its documentation](https://docs.julialang.org/en/v1/).  
+
+Julia can be downloaded [here](https://julialang.org/downloads/).
+
+### Basic Julia usage
+On the surface, Julia got many similarities to languages like MATLAB, Python, and R. 
+
+*Values* can be assigned to *variables* through the use of a `=` sign. Values (possibly stored in variables) can be used for most basic computations.
+```julia
+length = 2.0
+width = 4.0
+area = length*width
+```
+
+*Functions* takes some input (enclosed by `()`) and returns some output. E.g. the `min` function returns the minimum of two values
+```julia
+min(1.0,3.0)
+```
+
+A line of Julia code is not required to end with `;`, however, if it does, the output of that line is not displayed.
+```julia
+min(1.0,3.0);
+```
+
+Each Julia variable has a specific *type*, designating what type of value it is. While not directly required to use Catalyst, this is useful to be aware of. To learn the type of a specific variable, use the `typeof` function. More information about types can be [found here](https://docs.julialang.org/en/v1/manual/types/).
+```julia
+typeof(1.0)
+```
+
+Here, `Float64` denotes decimal-valued numbers. Integer-valued numbers instead are of the `Int64` type.
+```julia
+typeof(1)
+```
+
+Finally, we note that the first time some code is run in Julia, it has to be *compiled*. However, this is only required once per Julia session. Hence, the second time the same code is run, it runs much faster. E.g. try running this line of code first one time, and then one additional time. You will note that the second run is much faster.
+```julia
+(245^0.43) + exp(0.431) - log(124541) * tan(1.43) - sin(4.3);
```

Comment:
This is pretty instantaneous for me on 1.8.2 even with compilation. Maybe there is another example that has a more clear pause you could use?

Reply:
I was going to do something with big matrix multiplication, but I also wanted something where it was clear to the user what code was being run. I will have a think over the day, might do matrix multiplication, but point out that that is what is happening.